common/build.gradle

```diff
@@ -56,7 +56,7 @@ dependencies {
     api group: 'org.hdrhistogram', name: 'HdrHistogram', version: '2.1.9'
     api group: 'org.apache.commons', name: 'commons-collections4', version: '4.2'
     // We need this to avoid seeing SLF4J: Failed to load class "org.slf4j.impl.StaticLoggerBinder" on startup
-    api group: 'org.slf4j', name: 'slf4j-simple', version: '1.7.36'
+    api group: 'org.slf4j', name: 'slf4j-simple', version: '2.0.0'
```

Comment:
Without this change the cluster containers complain the logger is too old and they don't log anything